ParseUI-Login/src/main/res/values-ko/strings.xml

```diff
@@ -0,0 +1,43 @@
+<resources>
+
+    <string name="com_parse_ui_logo_content_description">Logo</string>
```

Comment:
We don't need this if it's not translated as it'll pull the default from `strings.xml`